server/routes.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const path = require('path');
+const getHtmlWithDecorator = require("./decorator");
+const buildPath = path.resolve(__dirname, "../build");
+const logger = require("./logger");
+
+const BASE_PATH = '/familie/alene-med-barn/soknad';
+
+const routes = () => {
+    const expressRouter = express.Router();
+
+    expressRouter.get(`${BASE_PATH}/internal/isAlive|isReady`, (req, res) =>
+        res.sendStatus(200)
+    );
+    expressRouter.use(BASE_PATH, express.static(buildPath, {index: false}));
+
+    expressRouter.use(/^(?!.*\/(internal|static)\/).*$/, (req, res) => {
+        getHtmlWithDecorator((path.join(buildPath, 'index.html')))
+            .then((html) => {
+                res.send(html);
```

Comment:
Skjønner ikke helt forskjellen på disse - min "antagelse" er jo at `express.static` blir kjørt når man laster siden og at man må legge på internal eller static for at den nederste skal kjøres. Isåfall skjønner jeg ikke hvordan dekoratøren blir lastet ettersom vi ikke bruker internal eller static når vi henter ut index-filen... 🤯

Reply:
Treffer ikke regexen på alt utenom internal || static?

Reply:
Jo, ser sånn ut, men trodde det ville blitt konflikt mellom linje 15 og linje 17 siden base-path treffer begge disse - og begge ender opp med `res.send` (som jo stopper "middelwaren" ?)

Reply:
Dette er for å støtte SPA